### DISCUSSION TITLE
Websocket support

### DISCUSSION THREAD

**Discussion by magbb** [kedacore/http-add-on#618] [General]
Hi! 

Thanks for this great add-on. 

Does the interceptor-proxy support websockets, or do I have to enable it somewhere? I have looked everywhere, but couldn't find anything. In my Python Streamlit (streamlit.io) webapp, which relies on websockets, I get a 403 forbidden for the websocket connections, everything else seems to work nicely.

Thanks in advance.

**Comment by wozniakjan** [comment]
https://github.com/kedacore/http-add-on/pull/835
